receiver/awscontainerinsightreceiver/internal/neuron/neuron_empty_metric_decorator.go

```diff
@@ -0,0 +1,159 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package neuron
+
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+const (
+	statusType     = "status_type"
+	errorType      = "error_type"
+	memoryLocation = "memory_location"
+	percentile     = "percentile"
+)
+
+var attributeConfig = map[string][]string{
+	NeuronExecutionStatus:                       {statusType},
+	NeuronExecutionErrors:                       {errorType},
+	NeuronRuntimeMemoryUsage:                    {memoryLocation},
+	NeuronExecutionLatency:                      {percentile},
+	NeuronCoreUtilization:                       {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationConstants:        {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationModelCode:        {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationSharedScratchpad: {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationRuntimeMemory:    {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+	NeuronCoreMemoryUtilizationTensors:          {neuronCoreAttributeKey, neuronDeviceAttributeKey},
+}
```

Comment:
How did you decide which metrics and attributes to include here?

Reply:
These are the only metrics that we publish to CW, for the attributes we looked at the EMF logs from our test cluster to get the minimum set of attributes for a EMF to be constructed